src/build/env.py

```diff
@@ -56,9 +58,15 @@ def install(self, requirements):  # type: (Iterable[str]) -> None
 class IsolatedEnvBuilder(object):
     """Builder object for isolated environments."""
 
+    _use_virtualenv = None  # type: Optional[bool]
+
     def __init__(self):  # type: () -> None
         self._path = None  # type: Optional[str]
 
+        if self._use_virtualenv is None:
+            unmet_deps = build.check_dependency('build[virtualenv]')
+            self.__class__._use_virtualenv = len(tuple(unmet_deps)) == 0
```

Comment:
This does not only check that virtualenv is installed; it checks that all dependencies are installed. Is that what we want to do?

Reply:
I think so. You don't want to run virtualenv when its dependencies are not satisfied, not?

Reply:
It checks that all of *build*'s dependencies are satisfied as well as virtualenv's.